### ISSUE TITLE
..\..\go\pkg\mod\github.com\!proton!mail\go-crypto@v0.0.0-20210428141323-04723f9f07d7\openpgp\ecdh\x25519.go:16:2: //go:build comment without // +build comment

### ISSUE THREAD

**Issue by qs3c** [micro/micro#2050]
After I ran my main.go file, it didn't register the service successfully and the following feedback appeared, what is wrong with this does anyone know?
`..\..\go\pkg\mod\github.com\!proton!mail\go-crypto@v0.0.0-20210428141323-04723f9f07d7\openpgp\ecdh\x25519.go:16:2: //go:build comment without // +build comment`

**Comment by qs3c** [comment]
Feedback like this doesn't even seem to be a reported error, leaving me clueless as to what went wrong.

**Comment by qs3c** [comment]
Not only running my own, but running the author's example gives the same feedback, why is that?

**Comment by qs3c** [comment]
I have resolved this issue and the problem is in line 16 in the `x22519.go` file under the folder `..\..\go\pkg\mod\github.com\!proton!mail\go-crypto@v0.0.0-20210428141323-04723f9f07d7\openpgp\ecdh`, as the feedback says.  
here is the problem：`import "golang.org/x/crypto"`
I added this sentence to go.mod: `replace golang.org/x/crypto v0.9.0 => github.com/golang/crypto v0.13.0`.
And then, I deleted all library caches `go clean --modcache` and re-downloaded it `go mod tidy` and it worked fine.
